src/server/src/Server.cpp

```diff
@@ -200,6 +212,37 @@ std::string Server::track_position(const double x, const double y, const double
     return "OK";
 }
 
+std::string Server::set_position_track_param(const double amax, const double vmax, const double jerk)
+{
+    State state = get_state();
```

Comment:
```suggestion
```

Reply:
See the comment above about the possibility not to allow the user changing the parameter if the state is different from `Idle`.

Reply:
let's check together

Reply:
have included the Idle part. Please verify.